gossip/src/duplicate_shred_handler.rs

```diff
@@ -171,10 +169,7 @@ impl DuplicateShredHandler {
         // Also skip frozen slots or slots with a different proof than me.
         match self.chunk_map.get(&slot) {
             Some(SlotStatus::Frozen) => false,
-            Some(SlotStatus::UnfinishedProof(slot_map)) => match slot_map.get(&data.from) {
-                None => true,
-                Some(proof_chunkmap) => proof_chunkmap.wallclock == data.wallclock,
-            },
+            Some(SlotStatus::UnfinishedProof(_)) => true,
```

Comment:
Yeah, that is fine.
A node should not gossip more than one duplicate for each slot;
If it erroneously or maliciously wants to send more than one, then it can also easily use the same `wallclock` for all of them. So relying on `wallclock` wouldn't be effective to filter those out.

Reply:
Sure, if that's the case we should really remove the wallclock field from the original message, waste of bytes anyway.

Reply:
The wallclock is used in gossip to for a node to update its older values.
That is legit use-case for wallclock in gossip.

Reply:
Wait, so we do assume wallclock to somewhat reflect real time in Gossip? But I assume that wallclock would be a general field across many Gossip messages, not piggybacked on every type of Gossip message, because most Gossip messages will need to figure out what value is older and needs to be overwritten?

Reply:
* If I want to update an entry (belonging to my node) in gossip I will have to attach a higher `wallclock` to the updated value.
* Other nodes will use this `wallclock` to deterministically and uniformly decide which version of the value _overrides_ another version in the gossip table.
* However just the fact that a value overrides another entry in the gossip table does not mean anything beyond that in terms of how much the value can be trusted or if the `wallclock` is true etc